libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -15,27 +15,26 @@ import {
   Output,
   QueryList,
   Renderer2,
+  RendererStyleFlags2,
   TemplateRef,
   ViewChild,
   ViewChildren,
 } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 
+import { KeyboardHandlerService } from '../../helpers';
+import { ButtonComponent } from '../button/button.component';
 import { CardComponent } from '../card/card.component';
 import { ItemComponent } from '../item/item.component';
 import { ListItemTemplateDirective } from '../list/list.directive';
+import { HorizontalDirection, PopoverComponent } from '../popover/popover.component';
```

Comment:
Unrelated to line 30 but had to add the comment somewhere. 

It seems like `popout` input property is undocumented. Should probably be added to API description.

Reply:
On the other hand it might be on purpose as it is only used internally. Might be worth following up on - if it is meant to be only used internally in Kirby a comment should however be added detailing this.

Reply:
Followed up on it - it seems like it should just be documented. 
Either create a new issue for it or just do it as part of this PR.